testing_and_setup/compass/ocean/global_ocean/WC14/init/define_base_mesh.py

```diff
@@ -0,0 +1,190 @@
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+import numpy as np
+import jigsaw_to_MPAS.mesh_definition_tools as mdt
+from jigsaw_to_MPAS.coastal_tools import signed_distance_from_geojson, \
+    mask_from_geojson
+from geometric_features import read_feature_collection
+import xarray
+import matplotlib
+import matplotlib.colors as colors
+matplotlib.use('Agg')
+
+
+def cellWidthVsLatLon():
+    """
+    Create cell width array for this mesh on a regular latitude-longitude grid.
+    Returns
+    -------
+       cellWidth : numpy.ndarray
+            m x n array, entries are desired cell width in km
+       lat : numpy.ndarray
+            latitude, vector of length m, with entries between -90 and 90,
+            degrees
+       lon : numpy.ndarray
+            longitude, vector of length n, with entries between -180 and 180,
+            degrees
+    """
+    # To speed up for testing, set following line to 1.0 degrees
+    dlon = 0.1
+    dlat = dlon
+    print('\nCreating cellWidth on a lat-lon grid of: {0:.2f} x {0:.2f} degrees'.format(dlon,dlat))
+    print('This can be set higher for faster test generation\n')
+    nlon = int(360. / dlon) + 1
+    nlat = int(180. / dlat) + 1
+    lon = np.linspace(-180., 180., nlon)
+    lat = np.linspace(-90., 90., nlat)
+    km = 1.0e3
+
+    print('plotting ...')
+    fig = plt.figure()
+    plt.clf()
+    fig.set_size_inches(10.0, 14.0)
+    mdt.register_sci_viz_colormaps()
+    #cmapBluesHalf = truncate_colormap(cmapIn='Blues', minval=0.0, maxval=0.7)
+
+    # Create cell width vs latitude for Atlantic and Pacific basins
+    QU1 = np.ones(lat.size)
+    EC60to30 = mdt.EC_CellWidthVsLat(lat)
+    EC60to30Narrow = mdt.EC_CellWidthVsLat(lat, latPosEq = 8.0, latWidthEq = 3.0)
+
+    # Expand from 1D to 2D
+    _, cellWidth = np.meshgrid(lon, EC60to30Narrow)
+    plot_cartopy(2, 'narrow EC60to30', cellWidth, '3Wbgy5')
+    plotFrame = 3
+
+    # global settings for regionally refines mesh
+    highRes = 14.0 #[km]
```

Comment:
This design is currently identical to the 12km WC12to60kmL60E3SMv2r01 in #555, except that the high resolution is set to 14km here.